pkgs/applications/science/electronics/kicad/default.nix

```diff
@@ -227,10 +229,19 @@ stdenv.mkDerivation rec {
 
         # link in the CLI utils
         (map (util: "ln -s ${base}/bin/${util} $out/bin/${util}") utils)
+
+        "runHook postInstall"
```

Comment:
any idea why this has to be added explicitly?

Reply:
Because you are overwriting the phase that normally runs those.